acceptance/tests/fixtures/bases/multiport-app/anyuid-scc-rolebinding.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Added a new multiport app-- this was much easier to read than trying to patch the existing static-server app. It just runs http-echo on two different ports with 2 different messages.